project/app/src/test/java/com/nexte/nexte/RankingScene/RankingPresenterTest.kt

```diff
@@ -187,7 +187,7 @@ class RankingPresenterTest {
     fun successCalculatePlayerLastGameCase4(){
         //prepare
         val todayDate = Date(2018, 5, 20)
-        val twoDaysAgo = Date(todayDate.year, todayDate.month, todayDate.day-2)
+        val twoDaysAgo = Date(todayDate.year, todayDate.month, 18)
```

Comment:
é uma solução paliativa, pois quando for dia 1-17 ele vai falhar, a solução melhor seria comentar esse teste como eu fiz na feature/390 e achar uma soluçao posteriar pra branch do case4.

Reply:
Mas bom trabalho